charts/stable/nextcloud/templates/_configmap.tpl

```diff
@@ -32,6 +32,19 @@ php-tune:
       pm.min_spare_servers = {{ .Values.nextcloud.php.pm_min_spare_servers }}
       pm.max_spare_servers = {{ .Values.nextcloud.php.pm_max_spare_servers }}
 
+opcache:
+  enabled: true
+  data:
+    opcache-recommended.ini: |
+      opcache.enable=1
+      opcache.save_comments=1
+      opcache.jit=1255
```

Comment:
I don't think we should expose those.
Maaaybe `jit` but still leaves a ton of room for "stupid" things.